pkg/crosscluster/logical/testdata/ldr_statements

```diff
@@ -76,22 +76,23 @@ ok
 
 show-insert table=products
 ----
-INSERT INTO [108 AS replication_target](id, name, unit_price, quantity, last_updated) VALUES ($1::INT8, $2::STRING, $3::DECIMAL(10,2), $4::INT8, $5::TIMESTAMP)
+INSERT INTO [108 AS replication_target](id, name, unit_price, quantity, last_updated) VALUES ($1::INT8, $2::STRING, $3::DECIMAL(10,2), $4::INT8, $6::TIMESTAMP)
 
+# NOTE: total_price and discount_price are not included since they are computed.[
 show-update table=products
 ----
-UPDATE [108 AS replication_target] SET id = $6::INT8, name = $7::STRING, unit_price = $8::DECIMAL(10,2), quantity = $9::INT8, last_updated = $10::TIMESTAMP WHERE ((((id IS NOT DISTINCT FROM $1::INT8) AND (name IS NOT DISTINCT FROM $2::STRING)) AND (unit_price IS NOT DISTINCT FROM $3::DECIMAL(10,2))) AND (quantity IS NOT DISTINCT FROM $4::INT8)) AND (last_updated IS NOT DISTINCT FROM $5::TIMESTAMP)
+UPDATE [108 AS replication_target] SET id = $7::INT8, name = $8::STRING, unit_price = $9::DECIMAL(10,2), quantity = $10::INT8, last_updated = $12::TIMESTAMP WHERE ((((id = $1::INT8) AND (name IS NOT DISTINCT FROM $2::STRING)) AND (unit_price IS NOT DISTINCT FROM $3::DECIMAL(10,2))) AND (quantity IS NOT DISTINCT FROM $4::INT8)) AND (last_updated IS NOT DISTINCT FROM $6::TIMESTAMP)
 
+# NOTE: total_price and discount_price are not included since they are computed.[
 show-delete table=products
 ----
-DELETE FROM [108 AS replication_target] WHERE ((((id IS NOT DISTINCT FROM $1::INT8) AND (name IS NOT DISTINCT FROM $2::STRING)) AND (unit_price IS NOT DISTINCT FROM $3::DECIMAL(10,2))) AND (quantity IS NOT DISTINCT FROM $4::INT8)) AND (last_updated IS NOT DISTINCT FROM $5::TIMESTAMP)
+DELETE FROM [108 AS replication_target] WHERE ((((id = $1::INT8) AND (name IS NOT DISTINCT FROM $2::STRING)) AND (unit_price IS NOT DISTINCT FROM $3::DECIMAL(10,2))) AND (quantity IS NOT DISTINCT FROM $4::INT8)) AND (last_updated IS NOT DISTINCT FROM $6::TIMESTAMP)
 
-# TODO(jeffswenson): BUG support computed columns in the primary key. Note that
-# the total_value is part of the primary key, yet it is not incldued in the
-# primary key CTE.
+# NOTE: total_price is not included because it is a computed column, but
+# discount_price is included because its part of the primary key.
```

Comment:
i think this comment is incorrect? i think you may need to add a case with a virtual column in the pk.

Reply:
oh, total_value is the computed column in the pk.